apps/system/js/notifications.js

```diff
@@ -403,44 +402,57 @@ var NotificationScreen = {
      */
     var dir = (detail.bidi === 'auto' || typeof detail.bidi === 'undefined') ?
       document.documentElement.dir : detail.bidi;
-
+    // As per the UX,in case for sms notification ,the notification title
+    // will be the count along with the 'new message' text,but currently in
+    // 'detail' object, name of the sender is coming. in other cases title text
+    // in 'detail' object is appropriate.
+    var cIndex = this.unReadNotificationIndex(manifestURL);
+    if (cIndex === -1) {
```

Comment:
I have done changes in that 60 lines of code as well, if i indent complete block of lines then it will be very hard to review the changes for David. Once David review this code and approve it then i will indent this block in any future lock screen commits. 

This is inline as per the second part from David's comments last time => "(I know I complained on an earlier version about unnecessary changes making the diff too big. But I think those were about accidental, incorrect indentation changes. In this case, the code must be indented in order to be legible, even if it adds 60 lines to your diff. (When something like that happens, you can always add a comment here on github letting me know that nothing has changed other than indentation.)"